sonarts-core/yarn.lock

```diff
@@ -3342,7 +3342,11 @@ lodash.templatesettings@^3.0.0:
     lodash._reinterpolate "^3.0.0"
     lodash.escape "^3.0.0"
 
-lodash@4.17.4, lodash@^4.14.0, lodash@^4.17.2, lodash@^4.17.4:
+lodash@4.17.10:
+  version "4.17.10"
+  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.10.tgz#1b7793cf7259ea38fb3661d4d38b3260af8ae4e7"
+
+lodash@^4.14.0, lodash@^4.17.2, lodash@^4.17.4:
   version "4.17.4"
```

Comment:
@andrea-guarino-sonarsource but we continue to use old version here. It's probably transitive dep

Reply:
Indeed, thanks! Fixed in 32991f02f4586053e64d4b8a1bb1272eadbda498